test-specifications/token-service/tokenPauseTransaction.md

```diff
@@ -0,0 +1,88 @@
+| commonTransactionParams | [json object](../commonTransactionParameters.md) | optional          |                               |
+
+### Output Parameters
+
+| Parameter Name | Type   | Description/Notes                                                                  |
+|----------------|--------|------------------------------------------------------------------------------------|
+| status         | string | The status of the submitted `TokenPauseTransaction` (from a `TransactionReceipt`). |
+
+## Property Tests
+
+### **Token ID:**
+
+- The ID of the token to pause.
+
+| Test no | Name                                                      | Input                                                                                   | Expected response                                                               | Implemented (Y/N) |
+|---------|-----------------------------------------------------------|-----------------------------------------------------------------------------------------|---------------------------------------------------------------------------------|-------------------|
+| 1       | Pauses a token                                            | tokenId=<CREATED_TOKEN_ID>, commonTransactionParams.signers=[<CREATED_TOKEN_PAUSE_KEY>] | The token pause succeeds and the token is paused.                               | N                 |
+| 2       | Pauses a token with no token ID                           |                                                                                         | The token pause fails with an INVALID_TOKEN_ID response code from the network.  | N                 |
+| 3       | Pauses a token without signing with the token's pause key | tokenId=<CREATED_TOKEN_ID>                                                              | The token pause fails with an INVALID_SIGNATURE response code from the network. | N                 |
+| 4       | Pauses a token that doesn't exist                         | tokenId="123.456.789"                                                                   | The token pause fails with an INVALID_TOKEN_ID response code from the network.  | N                 |
+| 5       | Pauses a token that is deleted                            | tokenId=<DELETED_TOKEN_ID>, commonTransactionParams.signers=[<DELETED_TOKEN_PAUSE_KEY>] | The token pause fails with an TOKEN_WAS_DELETED response code from the network. | N                 |
+| 6       | Pauses a token that is empty                              | tokenId=""                                                                              | The token pause fails with an SDK internal error.                               | N                 |
+| 7       | Pauses a token twice                                      | tokenId=<CREATED_TOKEN_ID>, commonTransactionParams.signers=[<CREATED_TOKEN_PAUSE_KEY>] | The token pause fails with an TOKEN_IS_PAUSED response code from the network.   | N                 |
+
```

Comment:
Shall we also add a scenario where the transaction is signed with the admin key?

Reply:
You mean instead of the pause key? Is that allowed behavior?

Reply:
Admin key cannot unpause; only the pause key can do that.  Admin key can change the keys (if the key is not disabled), and a few other tasks, but any task with its own key is denied to admin and dedicated solely to that key.

A test signed with admin and _expected to fail_ might be useful.

Reply:
Gotchya, added a test